clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -51,18 +51,19 @@ enum RewriteMethod {
     TOP_TERMS = 5;
 }
 
+// Low and high values for auto fuzziness. Defaults to low: 3 and high: 6 if both are unset. Valid values are low >= 0 and low < high
+message AutoFuzziness {
+    int32 low = 6; // Optional low distance argument.
+    int32 high = 7; // Optional high distance argument.
+}
+
 message FuzzyParams {
     int32 maxEdits = 1; // The maximum allowed Levenshtein Edit Distance (or number of edits). Possible values are 0, 1 and 2. Either set this or auto.
     int32 prefixLength = 2; // Length of common (non-fuzzy) prefix
     int32 maxExpansions = 3; // The maximum number of terms to match.
     bool transpositions = 4; // True if transpositions should be treated as a primitive edit operation. If this is false, comparisons will implement the classic Levenshtein algorithm. Default is true.
     AutoFuzziness auto = 5; // Auto fuzziness which determines the max edits based on the term length. AUTO is the preferred setting. Either set this or maxEdits.
 
-    // Optional low and high values for auto fuzziness. Defaults to low: 3 and high: 6 if both are unset. Valid values are low >= 0 and low < high
-    message AutoFuzziness {
-        int32 low = 6; // Optional low distance argument.
-        int32 high = 7; // Optional high distance argument.
-    }
```

Comment:
Will moving this message cause issues for existing clients? You can always declare the nested message as `FuzzyParams.AutoFuzziness`